requirements.txt

```diff
@@ -1,2 +1,3 @@
 netCDF4>=1.5.2
 numpy>=1.10.0
+geopandas>=0.6.2
```

Comment:
isn't pandas a requirement as well?

Reply:
It is installed as dependency for geopandas.

Reply:
I would still list it as a dependency since it's not only a transient dependency but a direct dependency as well.

Reply:
You are right, [done](https://github.com/emissions-api/sentinel5algorithms/pull/17/commits/3f01b51eb0bb0bad1bb9fa70a49c9f6b66b8feb8).